packages/office-ui-fabric-react/src/components/Dialog/Dialog.styles.ts

```diff
@@ -31,3 +33,30 @@ export const getStyles = (props: IDialogStyleProps): IDialogStyles => {
     ]
   };
 };
+
+export const getMergedDialogContentStyles = (
+  dialogContentProps?: IDialogContentProps,
+  dialogDraggableClassName?: string
+): IStyleFunctionOrObject<IDialogContentStyleProps, IDialogContentStyles> => {
+  const headerStyle = {
+    header: [
+      dialogDraggableClassName,
+      dialogDraggableClassName && {
+        cursor: 'move'
+      }
+    ]
+  };
+
+  if (dialogContentProps && dialogContentProps.styles) {
```

Comment:
I don't really follow this chunk of code. It seems wrong when:

1. we're returning a lambda inside of a styles function
2. we're calling `mergeStyleSets` manually rather than letting the helper hocs do that.

Reply:
@dzearing aren't both needed? 1) we are calling the given style function which not known until it is called (because the function we may get it created/altered by `DialogContent.base.tsx`)  AND 2) we need to call `mergeStyleSets` so that we can add the draggable cursor "for free"

Reply:
After discussion with some folks, this should be done in `DialogContent.styles.ts` along with the rest of the content styling.

Reply:
Then Dialog can just pass along a boolean like isDragging, which can be consumed by the styles to conditionally set the cursor.